blog/2015/04/osg-user-school.html

```diff
@@ -0,0 +1,40 @@
+---
+layout: blog
+root: ../../..
+author: Christina Koch
+title: Invitation to Millions of Compute Hours: Announcing the Open Science Grid User School
```

Comment:
You'll need to wrap the title in quotes, or the colon in it will confuse the parser:
    title: "Invitation to...: User School"